indent/pgsql.vim

```diff
@@ -44,10 +44,22 @@ set cpo&vim
 " IS is excluded, since it is difficult to determine when the
 " ending block is (especially for procedures/functions).
 let s:SQLBlockStart = '^\s*\%('.
-                \ 'if\|else\|elseif\|elsif\|'.
-                \ 'while\|loop\|do\|for\|'.
-                \ 'declare\|begin\|'.
-                \ 'case\|when\|merge\|exception'.
+                \ 'if\>.*\<then\|'.
```

Comment:
Can you please provide an example of what this improves over the current rule?

Reply:
Never mind, it's in the commit message :)

Reply:
:)